pom.xml

```diff
@@ -166,6 +226,7 @@
                 <groupId>org.projectlombok</groupId>
                 <artifactId>lombok</artifactId>
                 <version>1.18.18</version>
+                <scope>provided</scope>
```

Comment:
nit: I think the usual pattern for compile-only dependencies is `scope=provided` and `optional=true`. Note that the version (and the scope/optional) are now defined in the parent pom ([discovery-parent-pom](https://github.com/wikimedia/wikimedia-discovery-discovery-parent-pom)) and could be ignored here. To be honest, I'm not sure if it is better to rely on the parent pom or to duplicate the version and scopes.